src/Zigzag_persistence/doc/Intro_zigzag_persistence.h

```diff
@@ -0,0 +1,88 @@
+ * or removed one at a time and the returned persistence pairs / bars are indexed on the operation numbers.
+ * For example, if a cycle is born at operation number 6 and dies at operation number 7, it will output a bar starting
+ * at 6 and ending at 7.
+ * - @ref Filtered_zigzag_persistence and @ref Filtered_zigzag_persistence_with_storage are adding the notion of
+ * "filtration value" to @ref Zigzag_persistence. At each call, an operation can be associated to a filtration value,
+ * which will be used to index the returned bars instead (bars with new length 0 are then ignored). The two classes
+ * also have more flexible inputs (the boundaries do not have to be ordered, nor identified continuously
+ * from 0). The difference between both classes is on the way they manage the memory: @ref Filtered_zigzag_persistence
+ * removes systematically all unnecessary information and outputs a pair as soon it is closed, while
+ * @ref Filtered_zigzag_persistence_with_storage will store all informations about filtration values and bars until the
+ * end and output the pairs only when asked. Depending on the use and the length of the filtration, one will be more
+ * efficient than the other and vice versa.
+ *
+ * The implementation is based on the algorithm introduced in \cite zigzag.
+ *
+ * \subsection zigzaginterface Stream-like interface
+ * 
+ * As removals are possible in zigzag filtration, the maximal size of the complex does not depend on the length of the 
+ * filtration anymore. This makes it possible to build very long fine tuned filtrations with relatively small complexes
+ * which can be processed without overreaching memory space. For this purpose, it is possible to feed the module with
+ * information about the filtration "on the fly" to avoid loading the whole filtration at once. Information about the
+ * current barcode can be retrieved between any steps via callback methods.
+ * 
+ * \section zigzagexamples Examples
+ *
+ * \subsection zzminusage Minimalistic examples
+ *
+ * \li \gudhi_example_link{Zigzag_persistence,example_usage_zigzag_persistence.cpp} - A simple example to showcase how
+ * to use the @ref Zigzag_persistence class to compute a barcode.
+ * <details>
+ *   @include example_usage_zigzag_persistence.cpp
+ * </details>
+ * \li \gudhi_example_link{Zigzag_persistence,example_usage_filtered_zigzag_persistence.cpp} - A simple example to
+ * showcase how to use the @ref Filtered_zigzag_persistence class to compute a barcode.
+ * <details>
+ *   @include example_usage_filtered_zigzag_persistence.cpp
+ * </details>
+ * \li \gudhi_example_link{Zigzag_persistence,example_usage_filtered_zigzag_persistence_with_storage.cpp} - A simple
+ * example to showcase how to use the @ref Filtered_zigzag_persistence_with_storage class to compute a barcode.
+ * <details open>
+ *   @include example_usage_filtered_zigzag_persistence_with_storage.cpp
```

Comment:
I think it is possible to tell it to skip the comment at the beginning of the file (the part with the license, author, etc).

Reply:
If it isn't, we can just remove the comment from the example (have `#include` on the first line).

Reply:
It is indeed possible. It should be working now.